terraform/jenkins/README.md

```diff
@@ -15,7 +15,13 @@ terraform apply -input=false tfplan
 terraform destroy -auto-approve
 ```
 
-
+### Restore
+**_Important:_**
+terraform destroy needs to be done before restore
```

Comment:
life_cycle destroy before create can make a trick...

Reply:
add taint command in Readme to destroy force ec2 recreation with snapshot if